### DISCUSSION TITLE
Is it possible to deploy an app that consists of more than one container?

### DISCUSSION THREAD

**Discussion by oxa-klesti** [basecamp/kamal#1130] [General]
Hello,

We need to deploy a web app that consists of two different containers that listen to a different set of routes. 

Is this possible with Kamal?

Thanks

**Comment by tuladhar** [comment]
You can utilize Caddy as a reserve proxy to handle the routing.

**Reply by hipertracker** [reply]
nginx is simpler

**Comment by oxa-klesti** [comment]
How would the deployment work in this case?

I didn’t notice a multi service deployment option with Kamal.

**Comment by oxa-klesti** [comment]
It is already deployed using nginx. Can it be handled using Kamal?
